ipn/localapi/localapi.go

```diff
@@ -944,13 +941,13 @@ func authorizeServeConfigForGOOSAndUserContext(goos string, configIn *ipn.ServeC
 	if !slices.Contains([]string{"windows", "linux", "darwin"}, goos) {
 		return nil
 	}
```

Comment:
This is kinda awkward. I'd just write a switch and do the "return nil" in the default case and nothing in the `case "windows", "linux", "darwin"`